CineElorrietaMVC/src/main/java/Modelo/Utiles.java

```diff
@@ -2,9 +2,7 @@
 
 import java.math.BigDecimal;
 import java.math.RoundingMode;
-import java.text.SimpleDateFormat;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
+
 
 public class Utiles {
```

Comment:
Me parece que los metodos que estáis usando en utiles tendrían mejor acomodo en otros clases. Por ejemplo, entre otros, el metodo accionadoBotonAnnadirProducto tiene pinta de ir en el controlador que gestione dicho botón. Y, con la mayoria del resto de los metodos se podría decir tres cuartas partes de lo mismo. La clase utiles se debería de usar exclusivamente para metodos residuales sin acomodo en clase alguna.

Reply:
Realmente la mayoria de los metodos son funcionalidades, hemos cambiado el nombre para que sea mas representativo.

Reply:
Empezar por crear una nueva clase lista plato y pasad ahí los metodos que tengan encaje en dicha clase. Y, así sucesivamente con el resto

Reply:
Listo, Lo hemos dividido en funcionalidadPlatos y funcionalidadProductos.